src/main/kotlin/org/malachite/estella/services/TaskService.kt

```diff
@@ -41,11 +41,19 @@ class TaskService(
         }
 
 
-    fun getTasksByOrganizationUuid(organizationUuid: String) =
+    fun getTasksByOrganizationUuid(organizationUuid: String): List<TaskDto> =
```

Comment:
Ogólnie moglibyśmy operować chyba na UUID zamiast na Stringu jak pisałem niżej ale to trochę detal

Reply:
Bump